### PR TITLE
AppGWv2 Changes

### PR DESCRIPTION
**Pull request by akshaysngupta**
## Description

This PR contains changes in Application Gateway cmdlets.
- Zones, New Skus (Standard_v2, WAF_v2), EnableFIPS flag
- Help, Test, Changelog

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Network/ChangeLog.md [hunk 1]
```diff
@@ -22,6 +22,10 @@
 
 ## Version 6.2.0
 * Enable Traffic Analytics parameters on Network Watcher cmdlets
+* Updated below cmdlets for Application Gateway
```
Comment (cormacpayne) [inline src/ResourceManager/Network/ChangeLog.md]: @akshaysngupta please move the snippet you are adding here to underneath the `## Current Release` header above

File: src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs [hunk 2]
```diff
@@ -266,6 +276,16 @@ private PSApplicationGateway CreateApplicationGateway()
                 applicationGateway.EnableHttp2 = true;
             }
 
+            if (this.EnableFIPS.IsPresent)
+            {
+                applicationGateway.EnableFips = true;
+            }
+
+            if (this.Zones != null)
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs]: @akshaysngupta should this be `this.Zone` ?

Comment (akshaysngupta) [reply src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs]: https://github.com/Azure/azure-rest-api-specs/blob/c696e9c6a3125703d7b0690bf27440d672c6e503/specification/network/resource-manager/Microsoft.Network/stable/2018-04-01/applicationGateway.json#L1643

File: src/ResourceManager/Network/ChangeLog.md (hunk 1 above)
Comment (akshaysngupta) [reply src/ResourceManager/Network/ChangeLog.md]: corrected.

**Comment by MikhailTryakhov** [comment]
@akshaysngupta please fix issues in tests ASAP - I'll send branch to release after noon

**Pull request closed without merging**